lib/ansible/collections/__init__.py

```diff
@@ -8,7 +8,7 @@
 
 from ansible.module_utils._text import to_bytes
 
-B_FLAG_FILES = frozenset([b'MANIFEST.json', b'galaxy.yml'])
+B_FLAG_FILES = frozenset([b'MANIFEST.json', b'galaxy.yml', b'galaxy.yaml'])  # FIXME: this is inconsistent with what ansible-galaxy CLI does
```

Comment:
(unrelated)

Reply:
Will you be removing this code because it's unrelated. Sounds like it should be in a separate PR.

Reply:
exactly

Reply:
https://github.com/ansible/ansible/pull/72665